src/sqlfluff/rules/L018.py

```diff
@@ -99,9 +99,7 @@ def indent_size_up_to(segs):
                                     LintFix(
                                         "create",
                                         seg,
-                                        self.make_whitespace(
-                                            " " * (-indent_diff), seg.pos_marker
-                                        ),
+                                        self.make_whitespace(" " * (-indent_diff)),
```

Comment:
Just noticing this... does this mean lint rules don't need to provide a position for fixes anymore? If so, that's a really nice simplification for rule developers! 💯 🎉  I (and possibly other people) are probably guilty of just passing **some** value from some segment without much thought or understanding of the significance.

Reply:
yes yes yes 🎉 .

The logging actually warns people now when the *do* provide a position - because while you *can*, you really shouldn't. `_position_segments` handles all of that.